commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/UnicycleDrive.scala

```diff
@@ -9,6 +9,7 @@ trait UnicycleProperties {
   val maxForwardVelocity: Velocity
   val maxTurnVelocity: AngularVelocity
   val maxAcceleration: Acceleration
+  val initPosition: Point
```

Comment:
We still shouldn't have it in properties, because this is dependent on the auto mode chosen.